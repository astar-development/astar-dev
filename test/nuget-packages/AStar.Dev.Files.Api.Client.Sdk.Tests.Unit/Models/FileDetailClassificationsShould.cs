using AStar.Dev.Utilities;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

[TestSubject(typeof(FileDetailClassifications))]
public class FileDetailClassificationsShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FileDetailClassifications { ClassificationId = 1, Name = "MockName" }.ToJson().ShouldMatchApproved();
}
