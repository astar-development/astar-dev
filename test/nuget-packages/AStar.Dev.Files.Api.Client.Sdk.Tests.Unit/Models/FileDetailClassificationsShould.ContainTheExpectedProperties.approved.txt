{
  "classificationId": 1,
  "name": "MockName"
}